support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -44,14 +41,14 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
     context: Context,
     services: Services
   ): FutureHandlerResult = {
-    val nowJavaTime = () => OffsetDateTime.now
-    val todayJodaDate = () => LocalDate.now(DateTimeZone.UTC)
+    val now = () => DateTime.now(DateTimeZone.UTC)
+    val today = () => LocalDate.now(DateTimeZone.UTC)
```

Comment:
we still need a "today" and a "right now" in the end (unless we want to change the behaviour of the zuora sub deduplication)